src/containers/Body.module.css

```diff
@@ -0,0 +1,8 @@
+.dates {
+  box-sizing: border-box;

```

Comment:
sposterei questa proprietà direttamente nel file `index.css` in modo da applicarla a tutti gli elementi